### DISCUSSION TITLE
Does milvus 2.0 support GPU

### DISCUSSION THREAD

**Discussion by shanghaikid** [milvus-io/milvus#6225] [Q&A and General discussion]
Is milvus 2.0 support GPU, I don't see any info in [milvus.io](https://milvus.io)

**Comment by yhmo** [comment]
No, the 2.0 rc1 doesn't support GPU.

**Reply by shanghaikid** [reply]
will it support GPU ?  when ? could you update the doc?

**Reply by yhmo** [reply]
Will update the roadmap page: https://milvus.io/docs/v2.0.0/roadmap.md
GPU support is planned in v2.1
